pvtoolsSrc/pvput.cpp

```diff
@@ -333,34 +376,11 @@ int main (int argc, char *argv[])
             return 1;
         }
 
-        std::vector<std::string> values;
-        // copy values from command line
-        for (int n = 0; optind < argc; n++, optind++)
-            values.push_back(argv[optind]);
-
         Putter thework;
 
-        for(size_t i=0, N=values.size(); i<N; i++)
-        {
-            size_t sep = values[i].find_first_of('=');
-            if(sep==std::string::npos) {
-                thework.bare.push_back(values[i]);
-            } else {
-                thework.pairs.push_back(std::make_pair(values[i].substr(0, sep),
-                                                       values[i].substr(sep+1)));
-            }
-        }
-
-        if(!thework.bare.empty() && !thework.pairs.empty()) {
-            usage();
-            fprintf(stderr, "\nCan't mix bare values and field=value pairs\n");
-            return 1;
-
-        } else if(thework.bare.size()==1 && thework.bare[0][0]=='[') {
-            // treat plain "[...]" as "value=[...]"
-            thework.pairs.push_back(std::make_pair("value", thework.bare[0]));
-            thework.bare.clear();
-        }
+        // copy values from command line
+        for (int n = 0; optind < argc; n++, optind++)
```

Comment:
minor fyi.  The unused `n` could be omitted.  `for(; cond; var++) {}` is perfectly valid.

Reply:
Oh yes, that's right. I didn't notice `n` was unused in this case. Thanks!